sessions/js-fetch/star-wars-console/README.md

```diff
@@ -0,0 +1,23 @@
+# Fetch Star Wars API and Log to Console
+
+Fetching data is awesome, yet it can be frightening at the beginning. But don't worry, we are going to make you familiar with the syntax and it will be fine ;)
```

Comment:
```suggestion
Fetching data is awesome, yet it can be frightening in the beginning. But don't worry – we are going to help you familiarize yourself with the syntax of fetch and all will be fine ;)
```